app/views/similarItemCommodityCode.scala.html

```diff
@@ -0,0 +1,53 @@
+@*
+ * Copyright 2018 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import config.FrontendAppConfig
+@import uk.gov.hmrc.play.views.html._
+@import controllers.routes._
+@import models.SimilarItemCommodityCode
+@import models.Mode
+@import views.ViewUtils._
+
+@(appConfig: FrontendAppConfig, form: Form[_], mode: Mode)(implicit request: Request[_], messages: Messages)
+
+@main_template(
+    title = s"${errorPrefix(form)} ${messages("similarItemCommodityCode.title")}",
+    appConfig = appConfig,
+    bodyClasses = None) {
+
+    @helpers.form(action = SimilarItemCommodityCodeController.onSubmit(mode), 'autoComplete -> "off") {
+
+        @components.back_link()
+
+        @components.error_summary(form.errors)
+
+        @components.heading("similarItemCommodityCode.heading")
+
+        <ul class="list list-bullet">
+            <li>If your business operates in other countries and you're aware that they have a commodity code ruling, it is useful for us to know to align rulings</li>
+            <li>If you know of a similar item that has a commodity code ruling, it can help us to classify your goods</li>
```

Comment:
I may be wrong but bullet lists start with a lower case don't they? http://govuk-elements.herokuapp.com/typography/#typography-lists

Reply:
This content was copied from the prototype